code/script.js

```diff
@@ -28,26 +40,223 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
-
-  // This little thing makes the chat scroll to the last message when there are too many to
-  // be shown in the chat box
+  //Makes the chat scroll to the last message when there are many 
   chat.scrollTop = chat.scrollHeight
 }
-
-// A function to start the conversation
+//Clears the chat window
+const clearWindow = () => {
+  chat.innerHTML = `
+    <section class="bot-msg">
+      <img src="assets/bot.png" id="endBot" alt="Bot" />
+    </section>
+  `
+}
```

Comment:
Love this! It would be so nice with a restart button as well, in case the user changes their mind